services/api/src/routes/mfa.js

```diff
@@ -20,6 +20,15 @@ const checkPasswordVerification = (ctx, next) => {
   return next();
 };
 
+//XXX: reuse the FIXED_SCHEMA validator from the validation.js file?
```

Comment:
@andrewplummer any suggestions or just let it be for now ? its used in a different context than the user